### DISCUSSION TITLE
Is there a way to handle pagination with a XPATH web scraping feed ?

### DISCUSSION THREAD

**Discussion by Fluidetom** [FreshRSS/FreshRSS#6804] [Q&A] [answered]
Hi,
I've successfully defined a XPath web scraping feed to retrieve all job offers published on this site : https://www.reseau-idee.be/fr/offres-emploi.
I would like to be able to retrieve all jobs offers currently published, not only the first 12 on the first page.
For example, the page "2" is at https://www.reseau-idee.be/fr/offres-emploi?page=1.

Is there a way in FreshRSS to handle pagination when setting up an XPath feed?
I've already contacted the webmaster, and he replied he had no intention of enabling / setting up an RSS feed on this website. So I'm left with XPath, which works, but is limited due to the pagination in place.
Any idea?
TIA

**Comment by Alkarex** [comment, marked as answer]
Getting the newest 12 articles might be sufficient if you refresh sufficiently often compared to the speed of publication of new articles.
Otherwise it is relatively easy to do with
https://github.com/RSS-Bridge/rss-bridge

**Comment by Fluidetom** [comment]
Thanks.
I've finally decided to stick with the first 12 articles and let FreshRSS refresh it sufficiently often.
I would have liked to to get all existing job offers at once, but then I noticed some of the oldest ones were actually outdated and no longer valid.
So this method to regularly fetch the first 12 ones is eventually the right one for me.
